server/controller/src/main/java/ai/starwhale/mlops/configuration/bean/StorageProperties.java

```diff
@@ -0,0 +1,15 @@
+package ai.starwhale.mlops.configuration.bean;
+
+import ai.starwhale.mlops.storage.s3.S3Config;
+import lombok.Data;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+
+@Data
+@Component
```

Comment:
this annotation can be removed from here

Reply:
done